Rubberduck.Main/ComClientLibrary/Abstract/UnitTesting/IFakesProvider.cs

```diff
@@ -79,5 +79,61 @@ public interface IFakesProvider
         [DispId(17)]
         [Description("Configures VBA.DateTime.Date calls.")]
         IFake Date { get; }
+
+        [DispId(18)]
+        [Description("Configures VBA.Math.Rnd calls.")]
+        IFake Rnd { get; }
+
+        [DispId(19)]
+        [Description("Configures VBA.Iteraction.DeleteSetting calls.")]
```

Comment:
typo: Interaction (same for `SaveSetting`, `GetSetting` and `GetAllSettings`)

Reply:
Good catch, sending a fix